docs/source/borealis_processes.rst

```diff
@@ -21,6 +21,7 @@ Experiment Components
 ..  toctree::
     :maxdepth: 4
 
+    experiment_handler
```

Comment:
I disagree here. I think experiment_handler is a runtime process, it reads the experiment but it isn't used to write the experiment.